modules/elprep/filter/main.nf

```diff
@@ -0,0 +1,99 @@
+process ELPREP_FILTER {
+    tag "$meta.id"
+    label 'process_high'
```

Comment:
This seems extreme for a high performance BAM/SAM processing tool - but if in your experience it does require a lot of RAM it's fine.

Reply:
Yes, It is a bit extreme, but `elprep filter` reads the entire file into ram, so it uses a lot of memory. In exchange, it's very fast. All depends on the size of the input file though.